### PR TITLE
[OSJC-151] Replace ImageRepository with ImageStream

### PR DESCRIPTION
**Pull request by jcantrill**
@adietish merging

### CODE REVIEW

File: src/main/java/com/openshift/internal/restclient/builders/SourceDeploymentBuilder.java [hunk 1]
```diff
@@ -1,88 +0,0 @@
...
-
-/**
- * SourceDeploymentBuilder supports building OpenShift resources that
- * can be deployed by injecting a user's source into an image.  This builder
- * produces the resources required to build and deploy an image.
- * 
- * @author Jeff Cantrill
- */
-public class SourceDeploymentBuilder {
-
```
Comment (adietish) [inline src/main/java/com/openshift/internal/restclient/builders/SourceDeploymentBuilder.java]: @jcantrill what's replacing deployments/buildconfigs when using ImageStreams?

Comment (jcantrill) [inline src/main/java/com/openshift/internal/restclient/builders/SourceDeploymentBuilder.java]: @adietish I'm not sure I follow your question.  From what I can tell, ImageStream is, at the moment, a 1:1 replacement for ImageRepo

**Comment by adietish** [comment]
I'm asking since the PR is removing builder classes which seem to create deployment configs? ImageDeploymentBuilder and SourceDeploymentBuilder

**Comment by jcantrill** [comment]
I removed then since they were part of the POC and I don't see how we 
will use them going forward.  They exist in history or the old repo now 
if we really need them

On 04/21/2015 09:27 AM, André Dietisheim wrote:
> I'm asking since the PR is removing builder classes which seem to create
> deployment configs? ImageDeploymentBuilder and SourceDeploymentBuilder
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-restclient-java/pull/9#issuecomment-94793498>.
>

-- 
Jeff Cantrill
Senior Software Engineer, Red Hat Engineering
Red Hat
Office: 703-748-4420 | 866-546-8970 ext. 8162420
jcantril@redhat.com
http://www.redhat.com

**Pull request merged**